tests/integration_tests/security/test_vulnerabilities.py

```diff
@@ -185,6 +185,33 @@ def spectre_meltdown_reported_vulnerablities(
     }
 
 
+def check_vulnerabilities_on_guest(status):
+    """
+    There is a REPTAR exception reported on m6i.metal for
+    guest kernel versions 4.14/5.10/6.1 when spectre-meltdown-checker.sh
+    script is run inside the guest from below the tests:
+        test_spectre_meltdown_checker_on_guest and
+        test_spectre_meltdown_checker_on_restored_guest
+    The same script when run on host doesn't report the
+    exception which means the instances are actually not vulnerable to REPTAR.
+    The only reason why the script cannot determine if the guest
+    is vulnerable or not because Firecracker does not expose the microcode
+    version to the guest.
+
+    The check in spectre_meltdown_checker is here:
+        https://github.com/speed47/spectre-meltdown-checker/blob/master/spectre-meltdown-checker.sh#L6635-L6637
```

Comment:
I'd prefer to use the permanent link.
```suggestion
        https://github.com/speed47/spectre-meltdown-checker/blob/0f2edb1a71733c1074550166c5e53abcfaa4d6ca/spectre-meltdown-checker.sh#L6635-L6637
```